contracts/feature/Ownable.sol

```diff
@@ -11,7 +11,20 @@ import "./interface/IOwnable.sol";
 
 abstract contract Ownable is IOwnable {
     /// @dev Owner of the contract (purpose: OpenSea compatibility)
-    address public override owner;
+    address private _owner;
```

Comment:
i'm expecting to see a change in the interface as well but im not seeing it in this PR?

Reply:
Interface should remain the same. It has a function: `owner()`, which was previously overridden by the owner variable which was public.

Reply:
ah yes you're right